.dockerhub/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM scratch
+COPY config-lint /
+ENTRYPOINT ["/config-lint"] 
```

Comment:
What does this do?

Reply:
Just makes a clean image of the entire repo. I followed [goreleaser docs](https://goreleaser.com/customization/#Docker) on creating the docker image.

It builds an image that I can use with docker to run the example files against terraform.

Reply:
ok, thanks.